pom.xml

```diff
@@ -195,7 +195,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-war-plugin</artifactId>
-                <version>2.6</version>
+                <version>3.2.1</version>
```

Comment:
@rnugraha Where are you getting 3.2.1 from? As far as I can find, 2.6 is the latest.